src/tests/Mission.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Mission } from '../../__mocks__/components/MissionMock';
+
+describe('Mission', () => {
+  const component = (
+    <Mission
+      id="1"
+      mission="Moon"
+      description="the moon"
+    />
+  );
+
+  test('should match snapshot', () => {
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
- [ ] it would be better to add more tests to all the components like expecting the text and when you reserve a rocket and also when you are a active member on missions

Reply:
✅ As requested by the previous code reviewer please consider adding some more UNIT TESTS with JEST for MISSIONS to meet the project requirement! 💯

Reply:
Please, kindly implement this as stated earlier. You can [reference this tutorial](https://medium.com/@wahib.adiyatma17/unit-testing-on-react-app-with-jest-f74f97e62c55). 👍